package.json

```diff
@@ -14,8 +14,9 @@
     "start": "node ./server/server.js",
     "seed": "node ./seedDb.js",
     "docker": "npm run build:prod && npm run seed & npm run start",
-    "lint" : "eslint .",
-    "lint:fix" : "eslint . --fix"
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "fix": "truncate -s -1 dummydata2.json && echo ] >> dummydata2.json"
```

Comment:
Is this still needed with the closing ] in your seed generation script?

Reply:
No, I didn't even realize it was still there.